src/web/components/Links.tsx

```diff
@@ -169,10 +169,10 @@ export const Links = ({ userId }: Props) => {
             >
                 Search jobs
             </a>
-            <div className={seperatorHideStyles} />
+            <div data-print-layout='hide' className={seperatorHideStyles} />
 
             {userId ? (
-                <div className={linkStyles}>
+                <div data-print-layout='hide' className={linkStyles}>
```

Comment:
if the parent is hidden, wouldnt you not need to have data-print-layout='hide' in this child?